src/CPPDataMember.cxx

```diff
@@ -81,6 +81,26 @@ static PyObject* dm_get(CPPDataMember* dm, CPPInstance* pyobj, PyObject* /* kls
             Py_INCREF(dm->fDescription);
             return dm->fDescription;
         }
+
+        if (Cppyy::IsEnumConstant(dm->fScope)) {
+            // anonymous enum
+            long long llval = Cppyy::GetEnumDataValue(dm->fScope);
+            const std::string& enum_type = Cppyy::ResolveEnum(dm->fScope);
+            
+            PyObject* bval;
+            if (enum_type == "bool") {
+                bval = (bool)llval ? Py_True : Py_False;
+                Py_INCREF(bval);
```

Comment:
One comment I would have is that this also enables [test12_enum_scope](https://github.com/compiler-research/cppyy/blob/ace71c9414541973a3360b0c1efdecbf13c028d4/test/test_datatypes.py#L785) on OS X with cling. Maybe there is something there that can also enable this test on other platforms.